### PR TITLE
Mixer pallet tests

### PR DESCRIPTION
**Pull request by shekohex**

### CODE REVIEW

File: pallets/anchor/src/lib.rs [hunk 1]
```diff
@@ -220,8 +217,12 @@ pub mod pallet {
 }
 
 impl<T: Config<I>, I: 'static> AnchorInterface<T, I> for Pallet<T, I> {
-	fn create(creator: T::AccountId, depth: u8) -> Result<T::TreeId, DispatchError> {
-		T::Mixer::create(T::AccountId::default(), depth)
+	fn create(creator: T::AccountId, depth: u8, max_edges: u32) -> Result<T::TreeId, DispatchError> {
+		// FIXME: is that even correct?
```
Comment (drewstone) [inline pallets/anchor/src/lib.rs:221]: We will want to integrate some token pallet to begin handling deposit sizes and probably add it as a function argument. What do you think?

File: pallets/anchor/src/lib.rs [hunk 2]
```diff
@@ -342,8 +343,7 @@ impl<T: Config<I>, I: 'static> AnchorInspector<T, I> for Pallet<T, I> {
 		Ok(true)
 	}
 
-	/// Check if this anchor has this edge
 	fn has_edge(id: T::TreeId, src_chain_id: T::ChainId) -> bool {
-		EdgeList::<T, I>::contains_key(id, src_chain_id)
+		true
```
Comment (drewstone) [inline pallets/anchor/src/lib.rs:347]: Why?

**Review by drewstone — APPROVED** [review]
Looks good!

File: pallets/anchor/src/lib.rs (hunk 2 above)
Comment (shekohex) [reply pallets/anchor/src/lib.rs:347]: I guess this happened when I merged the branch.

Will fix that.

File: pallets/anchor/src/lib.rs (hunk 1 above)
Comment (shekohex) [reply pallets/anchor/src/lib.rs:221]: Yes, I agree, we need to handle this correctly.

Comment (shekohex) [reply pallets/anchor/src/lib.rs:221]: We could do that in another PR.

**Pull request merged**